modules/gitops-benefits-of-argo-rollouts.adoc

```diff
@@ -0,0 +1,26 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/argo-rollouts-for-progressive-delivery-of-deployments.adoc
+
+:_content-type: CONCEPT
+[id="gitops-benefits-of-argo-rollouts_{context}"]
+= Benefits of Argo Rollouts
+
+While managing and coordinating advanced deployment strategies in traditional infrastructure involves long maintenance windows, automation with advanced infrastructure like {product-title} and {gitops-title} can help reduce these windows. However, setting up these strategies still requires a substantial effort. With Argo Rollouts, you can simplify this process with enhancements to enable application teams to define their rollout strategy declaratively. Teams no longer need to define multiple deployments and services or create automation for traffic shaping and integration of tests. Using Argo Rollouts, you can encapsulate all the required definitions of the declarative rollout strategy to automate and manage the process.
+
+The following are some of the benefits of using Argo Rollouts as a default workload in {gitops-title}:
+
+* Automated progressive delivery as part of the {gitops-shortname} workflow
+* Advanced deployment capabilities
+* Leverage advanced deployment strategies such as blue-green or canary
+* Zero downtime updates for deployments
+* Fine-grained, weighted traffic shifting
+* Able to test without any new traffic hitting the production environment
+* Automated rollbacks and promotions
+* Manual judgment
+* Customizable metric queries and analysis of business key performance indicators (KPIs)
+* Integration with ingress controller and {SMProductName} for advanced traffic routing
+* Integration with metric providers for deployment strategy analysis
+* Usage of multiple providers
+
+With Argo Rollouts, users who are not doing progressive delivery get structure and guidelines to use progressive delivery in end-user environments without their teams needing to learn about traffic managers and infrastructure. With automated rollouts, the Operator takes care of the security and helps manage the resources, cost, and time effectively. Existing users who use Argo CD with security and automated deployments get feedback early in the process and avoid problems that impact them.
```

Comment:
> With Argo Rollouts, users who are not doing progressive delivery get structure and guidelines to use progressive delivery in end-user environments without their teams needing to learn about traffic managers and infrastructure. 

I think this is a bit hard to follow. Suggestion:

> With Argo Rollouts, users can more easily adopt progressive delivery in end-user environments. This provides structure and guidelines without requiring teams to learn about traffic managers and complex infrastructure.

The second sentence anthropomorphizes the Operator in its current form:

> With automated rollouts, the Operator _**takes care of**_ the security and _**helps**_ manage the resources, cost, and time effectively.

When revising, please keep the following in mind:
- Can you be specific about what "takes care of the security" means? 
- Can you name the Operator you're referring to? (the xyz Operator)